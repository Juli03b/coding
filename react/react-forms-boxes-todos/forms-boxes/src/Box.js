const Box = ({color="black", height, width, deleteBox, dataTestId}) => (
    <div style={{backgroundColor: color, height: height + "rem", width: width + "rem"}} data-testid={dataTestId}>
        <button onClick={deleteBox}>X</button>
    </div>
);

export default Box;