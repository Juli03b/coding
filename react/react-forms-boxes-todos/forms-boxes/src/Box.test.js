import { render } from "@testing-library/react";
import Box from "./Box";

it("Should render", () => {
    render(<Box />);
});

it("Should match snapshot", () => {
    const {asFragment} = render(<Box />);

    expect(asFragment()).toMatchSnapshot();
});
