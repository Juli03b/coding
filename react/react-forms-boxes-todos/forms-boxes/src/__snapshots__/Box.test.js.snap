// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<DocumentFragment>
  <div
    style="background-color: black;"
  >
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
